docs/howto/operate/new-cluster/new-cluster.md

```diff
@@ -0,0 +1,204 @@
+(new-cluster:new-cluster)=
+# Add a new Kubernetes cluster
+
+This guide will walk through the process of adding a new cluster to our [terraform configuration](https://github.com/2i2c-org/infrastructure/tree/HEAD/terraform).
+
+```{attention}
+Currently, we do not deploy clusters to AWS using terraform.
+Please see [](new-cluster:aws) for AWS-specific deployment guidelines.
+```
+
+## Cluster Design
+
+This guide will assume you have already followed the guidance in [](/topic/cluster-design) to select the appropriate infrastructure.
+
+## Creating a Terraform variables file for the cluster
```

Comment:
```suggestion
## Create a Terraform variables file for the cluster
```

I think in general we go with "imperative" (create) over gerund-y verbs (creating)...not super opinionated about that though